templates/field/field--field-ucb-person-links.html.twig

```diff
@@ -19,7 +19,7 @@
 			<li{{item.attributes.addClass(classes)}}>
 
 				{% set linkTitle = item.content['#title'] %}
-				{% set linkUrl = item.content['#url']|render|striptags|trim|upper  %}
```

Comment:
Getting rid of the |upper filter means that we will never match against any of the Social Media Site checks below.